service/src/main/kotlin/fi/espoo/evaka/placement/PlacementPlanService.kt

```diff
@@ -89,7 +89,10 @@ class PlacementPlanService(private val asyncJobRunner: AsyncJobRunner<AsyncJob>,
                         PlacementType.PRESCHOOL_DAYCARE,
                         PlacementType.PREPARATORY_DAYCARE ->
                             FiniteDateRange(
-                                startDate,
+                                maxOf(
+                                    minStartDate,
+                                    form.preferences.connectedDaycarePreferredStartDate ?: startDate
+                                ),
```

Comment:
Shouldn't this be applied also to `PRESCHOOL_CLUB` below?

Reply:
Yes, I verified the occurrence of the problem for preschool club and applying the same logic fixes the start date